### ISSUE TITLE
Steam Workshop

### ISSUE THREAD

**Issue by liveskai** [Ezz-lol/boiii-free#93]
### Duplicate Issue Check

- [X] I have searched the list of issues on this repository. I confirm that this is not a duplicate of any open or closed issue(s).

### Your feature request or suggestion:

Unable to read steam workshop mod, need to create MODS folder in game directory to copy files
![image](https://github.com/Ezz-lol/boiii-free/assets/54350655/97718333-e25d-403f-b3a5-5481901c1d66)

**Comment by AttarGangster2** [comment]
whenever i update boiii client to 1.0.2, why it reverts back to 1.0.1. 
is this a bug? or some kind downgrading the client for a reasons?

**Comment by ZolarZ** [comment]
> ### Duplicate Issue Check
> 
>     * [x]  I have searched the list of issues on this repository. I confirm that this is not a duplicate of any open or closed issue(s).
> 
> 
> ### Your feature request or suggestion:
> 
> Unable to read steam workshop mod, need to create MODS folder in game directory to copy files ![image](https://private-user-images.githubusercontent.com/54350655/312061897-97718333-e25d-403f-b3a5-5481901c1d66.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc2NTc0MDUsIm5iZiI6MTcxNzY1NzEwNSwicGF0aCI6Ii81NDM1MDY1NS8zMTIwNjE4OTctOTc3MTgzMzMtZTI1ZC00MDNmLWIzYTUtNTQ4MTkwMWMxZDY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA2VDA2NTgyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRiNmVkZGE3MTU0ZjUyYjdmM2ZkNGJiNjUyZjhmNjY2MWFlMTFhYzdlYTI2ZTIyMjM0ZjllNGNkN2E5MjM5N2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fgHUMnEdw4p-JUlruYjsLotsYIZYh05qRn9wfMBglko)

Set steam to offline mode before starting boiii until its fixed
